app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @users = [User.find(params[:id])]
+    @posts = @users[0].recent_posts
```

Comment:
- [ ] - [ ] [Optional] It would be much better professionally if you called the recent posts method inside `user_show` view instead of calling it here.

```suggestion
    @user = User.find(params[:id])
    @posts = @users.posts
```